### DISCUSSION TITLE
cannot use the name `letters` or `letter`

### DISCUSSION THREAD

**Discussion by liigo** [alpinejs/alpine#2223] [7. Bugs (archived)] [answered]
```html
<ul x-data="{ letters: ['a','b','c'] }">
  <template x-for="letter in letters">
    <li x-text="letter"></li>
  </template>
</ul>
```

**Comment by danddanddand** [comment, marked as answer]
This is a bug. I tested a small patch that fixes it so I'll make a PR. https://codepen.io/danddanddand/pen/YzxyEBo

**Reply by SimoTod** [reply]
👍 You can put the space once after the parenthesis to save 2 characters

**Reply by danddanddand** [reply]
> 👍 You can put the space once after the parenthesis to save 2 characters

Oh ya, will do

**Reply by danddanddand** [reply]
Here's the PR https://github.com/alpinejs/alpine/pull/2224
